tls/s2n_tls13_handshake.c

```diff
@@ -235,6 +233,9 @@ int s2n_tls13_handle_handshake_secrets(struct s2n_connection *conn)
     GUARD(s2n_tls13_derive_finished_key(&secrets, &server_hs_secret, &server_finished_key));
     GUARD(s2n_tls13_derive_finished_key(&secrets, &client_hs_secret, &client_finished_key));
 
+    /* since early secrets have been computed, PSKs are no longer needed and can be cleaned up */
```

Comment:
We should probably change this comment now that we're not doing this calculation right after early secrets.

Reply:
We still are cleaning up the PSKs earlier than `s2n_connection_free`  as early secrets are computed and we no longer need them, we just need to do it at a later stage within the same function.

Reply:
We still are cleaning up the PSKs secrets earlier than s2n_connection_free as early secrets are computed and we no longer need them, we just need to do it at a later stage within the same function. Will update the comment to be more clearer.